### DISCUSSION TITLE
How to use the Prompt Styles Selector?

### DISCUSSION THREAD

**Discussion by UnclePsymon** [WASasquatch/was-node-suite-comfyui#255] [Q&A] [answered]
Hey there! I want to use the Prompt Styles Selector node, but i can't figure out where to connect it to. Where do the positive_string and the negative_string noodles go? Thanks :)

**Comment by UnclePsymon** [comment, marked as answer]
Got it! Text to input with the CLIP Text Encode node

**Reply by WASasquatch** [reply]
Yep, just convert text field to a input. Or can use text to conditioning node
